pom.xml

```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <!-- DSpace Version Information (supported version of DSpace) -->
-        <dspace.version>[3.0,4.0)</dspace.version>
+        <dspace.version>[3.0,6.0)</dspace.version>
```

Comment:
@bbranan : It appears that Travis CI doesn't like the new version range. It looks to me like it's trying to pull down `6.0-rc4` (which is how we tag "6.0 Release Candidate 4") and use it.  So, it's matching stuff in the 6.0 range of code, which it shouldn't be doing, and that causes errors.

Maybe we should try specifying a 5.x version here.  Currently, the latest in 5.x is v5.10.  So, we could either pin this at 5.10 (e.g. `[3.0,5.10]`), or set it to something larger just in case (e.g. `[3.0,5.20]`).